stylus/ui-app/forms.styl

```diff
@@ -11,9 +11,11 @@ checkbox-size = 20px
 
 $form
     .coz-form
-        position relative
         margin   1em 0 .5em
 
+    .coz-form
+    .coz-form-group
```

Comment:
Well, the use of flexbox on the form elements force us to stack labels and inputs, but sometimes an "input" may be composed of many elements, and I would like to group these elements into a div that flexes the same way than `coz-form`. Does it make sense to you?